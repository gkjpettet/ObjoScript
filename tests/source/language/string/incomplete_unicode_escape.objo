// Expect error: Incomplete Unicode escape sequence.
"\u00"