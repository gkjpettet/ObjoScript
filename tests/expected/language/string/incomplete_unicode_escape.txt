Incomplete Unicode escape sequence. Expected 4 hex digits after `\u`.